cli/generators/crud/crudGenerator.js

```diff
@@ -57,4 +57,43 @@ module.exports = class extends Generator {
       }
     );
   }
+
+  updateRoutes(templatePath, destinationPath, data) {
+    let fs = this.fs;
+    ejs.renderFile(templatePath, data, (err, renderedTemplate) => {
+      if(err) throw err;
+
+      fs.copy(
+        destinationPath,
+        destinationPath,
+        {
+          process: (content) => {
+            var tree = program(content.toString(), null, {sourceType: 'module', plugins: { jsx: true } });
+            var currentRoutes = tree.var('defaultExport').value().children;
+            var currentImports = tree.body.node;
+
+            var renderedTemplateTree = program(renderedTemplate, null, {sourceType: 'module', plugins: { jsx: true } });
+            var newRoutes = renderedTemplateTree.var('defaultExport').value().children;
+            var newImports = renderedTemplateTree.body.node.filter(function(node) {
+              return node.type === 'ImportDeclaration';
+            });
+
+            newImports.forEach(function(newImport) {
+              currentImports.splice(4, 0, newImport);
```

Comment:
It's for insert after the 4th current import. But now that you mention, I can add it at the beginning. I will, change it, thanks!